pymemcache/test/test_client.py

```diff
@@ -131,9 +131,21 @@ def test_set_success(self):
         result = client.set(b'key', b'value', noreply=False)
         assert result is True
 
+        # unit test for encoding passed in __init__()
+        client = self.make_client([b'STORED\r\n'], encoding='utf-8')
+        result = client.set(b'key', b'value', noreply=False)
+        assert result is True
+
     def test_set_future(self):
         client = self.make_client([b'STORED\r\n'])
-        result = client.set(newbytes(b'key'), newbytes(b'value'), noreply=False)
+        result = client.set(newbytes(b'key'), newbytes(b'value'),
+                            noreply=False)
```

Comment:
I think, if you move `noreply` back to the pervious line as it was before, we'll get a cleaner diff here (and there isn't a reason to break this line here).